src/blacklist/5_0_0/sd/blacklistDiscovery/blacklistDiscovery_sd.tex

```diff
@@ -0,0 +1,282 @@
+
+The \textbf{blacklistDiscovery} service contains the following operations:
+
+\begin{itemize}
+    \item \textit{lookup} returns blacklist entries in force that apply to the requester;
+    \item \textit{check} returns if a system name is on the blacklist.
+\end{itemize}
+
+The rest of this document is organized as follows.
+In Section \ref{sec:functions}, we describe the abstract message operations provided by the service.
+In Section \ref{sec:model}, we end the document by presenting the data types used by the mentioned operations.
+
+\subsection{How This Service Is Meant to Be Used}
+In some use cases, it might be important to get some information about the state of the blacklist. An application system can use the \textbf{blacklistDiscovery} service's operations to ask if a system is banned from the Local Cloud, even if the question applies to the requester itself. In the latter case, it is also possible to query the entries that cause the ban.
+
+\subsection{Important Delimitations}
+\label{sec:delimitations}
+
+The requester has to identify itself to use any of the operations.
+
+\subsection{Access policy}
+\label{sec:accesspolicy}
+
+Available for anyone within the Local Cloud.
+
+\newpage
+
+\section{Service Operations}
+\label{sec:functions}
+
+This section describes the abstract signatures of each operations of the service. The \textbf{blacklistDiscovery} service is used to \textit{lookup} for blacklist entries and to \textit{check} if another system is on the blacklist.
+In particular, each subsection names an operation, an input type and one or two output types (unsuccessful operations can return different structure), in that order.
+The input type is named inside parentheses, while the output type is preceded by a colon. If the operation has two output types, they are separated by a slash.
+Input and output types are only denoted when accepted or returned, respectively, by the operation in question. All abstract data types named in this section are defined in Section 3.
+
+\phantomsection
+\fsubsection{lookup}{Identity}{BlacklistEntryListResponse}{}{ErrorResponse}
+
+Operation \textit{lookup} returns the blacklist entries that are in force and apply to the system that the provided identity belongs to. This is the only operation that is available for a system even if it's blacklisted. The input operation data must meet the following criteria:
+
+\begin{itemize}
+    \item It is not possible with this operation to lookup for other system's entries.
+    \item It is not possible with this operation to lookup for records that are expired or inactive.
+\end{itemize}
+
+\fsubsection{check}{CheckRequest}{CheckResponse}{}{ErrorResponse}
+
+Operation \textit{check} returns if an arbitrary system is on the blacklist.
+
```

Comment:
Here, in the input data criteria you can copy the relevant things about system names (naming convention, allowed characters, max length) from one of the other documents.